lib/govuk_app_config/error.rb

```diff
@@ -0,0 +1,15 @@
+require "sentry-raven"
+require "govuk_app_config/statsd"
+
+module GOVUK
+  module Error
```

Comment:
This is a cool namespace, which makes me think it's possible there might be conflicts.